lib/conjur/command/hosts.rb

```diff
@@ -153,7 +159,7 @@ def self.host_layer_roles host
       end
     end
 
-    hosts.desc "[Deprecated] Enroll a new host into conjur"
+    hosts.desc "Enroll a new host into conjur [DEPRECATED]"
```

Comment:
I skipped adding it here because it's already using `hide_docs(c)`, just normalized the DEPRECATED text